src/components/Newsletter/Newsletter.js

```diff
@@ -211,6 +201,21 @@ class Newsletter extends React.Component {
           {this.state.errorMessage && (
             <InputError>{this.state.errorMessage}</InputError>
           )}
+          <InputHolder errMessageOpen={errMessageOpen}>
+            <input
+              type="checkbox"
+              value="wurhhh"
+              id="wurhhh"
+              name="cm-ol-wurhhh"
+              onChange={() => this.setState({ checked: !this.state.checked })}
+              checked={this.state.checked}
+            />
+            <label htmlFor="checkbox">
+              {' '}
+              Also send me <b>daily news</b> with the latest data and scientific
+              findings on COVID{' '}
+            </label>
```

Comment:
nit: why do we have spaces before and after? we can keep them if that makes spacing easier :)

Reply:
ooo im not sure. those were already there. ill nix them